incubator/consul/values.yaml

```diff
@@ -0,0 +1,208 @@
+## Bitnami Osclass image version
+## ref: https://hub.docker.com/r/bitnami/osclass/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/consul
+  tag: 1.0.6
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: Always
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistrKeySecretName
+
+## Kubernetes service type
+serviceType: ClusterIP
+
+## Consul replicas
+replicas: 3
+
+## Consul service ports
+httpPort: 8500
+rpcPort: 8400
+serflanPort: 8301
+serverPort: 8300
+consulDnsPort: 8600
```

Comment:
should we instead add a service key, as such:

```
service:
  type: ClusterIP
  ports:
    http: 8500
    rpc: 8400
    serflan: 8301
    server: 8300
    consulDns: 8600
```

the ui port will have be moved out of here. wdyt?